### PR TITLE
[READY] Luxury shuttle charges bank cards

### PR DESCRIPTION
**Pull request by cacogen**
:cl: cacogen
add: Luxury shuttle now attempts to charge humans' bank accounts in addition to taking held and dragged money
add: Luxury shuttle now attempts to return change to humans' bank accounts
add: You can now pay for the luxury shuttle in instalments (mostly useful for handless mobs)
tweak: Luxury shuttle ticket booth messages are now spoken, so you can spend loudly
/:cl:

### Other changes:
- Gives the ticket booth a unique name and description
- Gives humans a proc to return their bank account datum, or false if they don't have one

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
### Why:

1. Allows handless mobs to pay with denominations smaller than 500 credits
3. Ticket booth talks because it's better.

### What else I'd like to add:

- [x] Gets money straight from ID
- [ ] ~~Change proc?~~

This originally gave tickets that could be stolen from other players but Goof's economy pull means most people will have the money to board now.

### CODE REVIEW

**Comment by ninjanomnom** [comment]
Please only open prs which build on code in the master branch. Wait for the other pr to be merged before opening a further pr building on that. Ask any maintainer to reopen this later

**Comment by cacogen** [comment]
It duplicates the code from the previous PR

**Comment by cacogen** [comment]
@ninjanomnom Old one got merged so reopen if you would

**Comment by ninjanomnom** [comment]
Ask in the coderbus irc to get it open faster, that's where the maintainers hang out.

**Comment by Iamgoofball** [comment]
this should work exclusively with the bank card, we don't want people using other forms of money

they can convert it to their bank card by just smashing the ID with it if they want to

**Comment by cacogen** [comment]
I'm going to keep it so dragged money works so simple mobs can access the shuttle, but I can remove it looking for held money.

**Review by ninjanomnom — APPROVED** [review]
I hate how istype-ey this proc is but this seems fine. At some point we really should refactor how the money is counted

**Comment by ninjanomnom** [comment]
I don't see a good reason why

**Comment by Iamgoofball** [comment]
Unifying all the money systems. We can just make it easy to slam all these other types of cash into your account, so that we only have to just do a check against the account instead of all these istype checks.

**Pull request merged**
